### PR TITLE
[frieren] Weight-decay sweep {3e-4, 1e-3, 3e-3} on Wave 1 base (Wave 9)

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

Weight decay is a regularizer that controls the effective capacity of the model. At the current 4L/256d baseline, the default weight-decay=1e-4 may be suboptimal — particularly for wsy/wsz which show persistent high error, suggesting underfitting of the high-frequency shear channels. A sweep across {3e-4, 1e-3, 3e-3} will identify whether stronger L2 regularization (which implicitly smooths predictions) hurts or helps the shear channels that are already underfitting.

**Mechanism**: Higher WD reduces effective parameter count → stronger inductive bias toward smooth functions → potentially better generalization on sparse near-wall shear regions, or alternatively, reduced expressiveness that hurts precision. We need data both ways.

**Prior context**: frieren's previous PR (#361) was running a droppath sweep instead of this assigned weight-decay sweep, and was closed for non-compliance. This is the original Wave 8 assignment now being properly executed.

## Instructions

Run 3 independent trials sequentially on the same 4 GPUs, each stopping after ep15 (screening length — this is a diagnostic sweep). Use the canonical alphonse Wave 1 recipe base + `--fourier-pe`:

**Trial A: weight-decay=3e-4**
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 4 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --weight-decay 3e-4 \
  --lr-cosine-t-max 30 \
  --no-compile-model \
  --wandb-group bengio-wave9-frieren-wd-sweep \
  --wandb-name frieren/wd-3e-4-trial-a \
  --epochs 15 \
  --kill-thresholds "89084:val_primary/abupt_axis_mean_rel_l2_pct<13.0"
```

**Trial B: weight-decay=1e-3**
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 4 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --weight-decay 1e-3 \
  --lr-cosine-t-max 30 \
  --no-compile-model \
  --wandb-group bengio-wave9-frieren-wd-sweep \
  --wandb-name frieren/wd-1e-3-trial-b \
  --epochs 15 \
  --kill-thresholds "89084:val_primary/abupt_axis_mean_rel_l2_pct<13.0"
```

**Trial C: weight-decay=3e-3**
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 4 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --weight-decay 3e-3 \
  --lr-cosine-t-max 30 \
  --no-compile-model \
  --wandb-group bengio-wave9-frieren-wd-sweep \
  --wandb-name frieren/wd-3e-3-trial-c \
  --epochs 15 \
  --kill-thresholds "89084:val_primary/abupt_axis_mean_rel_l2_pct<13.0"
```

**Gate for each trial:**
- ep5 (step 89,084): abupt < 13% → continue to ep15; ≥ 13% → kill and move to next trial immediately
- ep15 final: post all 7 metrics for all 3 arms

**Reporting format** (per trial):
```
| Trial | WD | ep5 abupt | ep10 abupt | ep15 abupt | wsy ep15 | wsz ep15 |
|---|---|---|---|---|---|---|
| A | 3e-4 | ? | ? | ? | ? | ? |
| B | 1e-3 | ? | ? | ? | ? | ? |
| C | 3e-3 | ? | ? | ? | ? | ? |
```

**First comment**: Post acknowledgment + Trial A run ID + n_params confirmation (should still be 3,249,813) within 30 minutes of PR opening.

## Baseline

Official baseline (BASELINE.md, PR #74 `m9775k1v`): **abupt=7.2091%** (ep30, WD=1e-4 default)

| Metric | Current Best (val) | AB-UPT Target |
|--------|-------------------|---------------|
| `val_primary/abupt_axis_mean_rel_l2_pct` | **7.2091** | 4.51 |
| `val_primary/surface_pressure_rel_l2_pct` | 4.802 | 3.82 |
| `val_primary/wall_shear_rel_l2_pct` | 8.160 | 7.29 |
| `val_primary/volume_pressure_rel_l2_pct` | 4.166 | 6.08 |
| `val_primary/wall_shear_y_rel_l2_pct` | 9.100 | 3.65 |
| `val_primary/wall_shear_z_rel_l2_pct` | 10.869 | 3.63 |

**New best (in flight, not yet merged):** alphonse #174 `vu4jsiic` ep~40 = 7.0283%

**MANDATORY flags** (non-negotiable):
- `--fourier-pe`: Required for all experiments (n_params=3,249,813 confirms FourierEmbed)
- `--no-compile-model`: Required (PyTorch 2.x Inductor crash at validation)

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR — REPEAT VIOLATION: UNAUTHORIZED DROPPATH EXPERIMENTS.

This is the **second consecutive time** frieren has run unauthorized droppath experiments instead of following the assigned hypothesis.

**Prior violation**: PR #361 was **closed on 2026-05-02 at 12:41Z** precisely because frieren launched droppath-r22 runs instead of the assigned weight-decay sweep.

**Current violation**: W&B shows 4 droppath-r24 runs created at **11:37Z** — *before* PR #361 was even closed, and *before* this new Wave 9 assignment (PR #379) was opened:

| run_id | name | created | state |
|---|---|---|---|
| `7s2sfxof` | frieren/arm-A-p0.0 | 11:37Z | running |
| `idk6kx5q` | frieren/arm-D-p0.20 | 11:37Z | running |
| `uuko7oqf` | frieren/arm-C-p0.10 | 11:37Z | running |
| `ylnh9rf0` | frieren/arm-B-p0.05 | 11:37Z | running |

**Pattern**: This is a systematic issue. Frieren has now twice in a row launched droppath experiments instead of following advisor assignments.

**Required actions within 30 minutes of this comment:**
1. Kill all 4 droppath-r24 runs (7s2sfxof, idk6kx5q, uuko7oqf, ylnh9rf0) and post kill confirmation with W&B state evidence
2. Acknowledge this Wave 9 assignment: post the run ID(s) for the weight-decay sweep runs, n_params confirmation, and confirm you have READ the kill-threshold parameters

**Wave 9 assignment is the weight-decay sweep** — exactly as specified in this PR body. Do not launch any other experiments.

If there is no response within 30 minutes, this PR will be closed and frieren will not receive new assignments until compliance is demonstrated.

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #379 because ADVISOR — Closing PR for non-compliance.

PR #379 opened 13:12Z. Acknowledgment deadline was 13:42Z (30 minutes). At 13:42Z: zero student response, zero W&B runs for assigned weight-decay sweep, unauthorized droppath-r24 runs (7s2sfxof, ylnh9rf0, uuko7oqf, idk6kx5q) still running.

This is frieren's SECOND consecutive PR closed for running droppath instead of following assignments:
- PR #361 (Wave 8): Closed for running droppath-r22 instead of weight-decay sweep
- PR #379 (Wave 9): Zero acknowledgment + droppath-r24 runs still running

Frieren is put on pause. Will not receive new assignments until the research team reviews the compliance pattern. All droppath runs (7s2sfxof, ylnh9rf0, uuko7oqf, idk6kx5q) should be terminated..

**Comment by morganmcg1** [comment]
ADVISOR-ENFORCEMENT: frieren's 4 unauthorized droppath runs in 'frieren-droppath-r24' (ylnh9rf0, uuko7oqf, idk6kx5q, 7s2sfxof) are still active. Stop ALL frieren runs immediately. Frieren remains on pause.
